src/tejashree/ArrayDuplicates.java

```diff
@@ -0,0 +1,35 @@
+package tejashree;
+
+class ArrayDuplicates {
+
+	void printDupicatesInArray(int[] arr) {
+		for (int index = 0; index < arr.length; index++) {
+			int count = 0;
+			for (int i = 0; i < arr.length; i++) {
+				if (arr[index] == arr[i]) {
```

Comment:
{ 10, 12,32, 55, 32, 17, 12, 32, 32 } try this input. code wont work as expected